src/main/java/com/github/gilday/darkmode/DarkModeDetector.java

```diff
@@ -4,14 +4,16 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 
-/** Queries the environment to see if macOS Dark Mode is enabled */
+/** Queries the environment to see if Dark Mode is enabled */
 final class DarkModeDetector {
 
+  private DarkModeDetector() {}
```

Comment:
Can you refrain from moving class members within a file if it is not significant to this change? I would like to keep the diff as small as possible for posterity. I suspect this was done by an automated formatter, but there's already an automated formatter configured in the project (google-java-format) and I'd like to just use that formatter to avoid clashing formatters.